src/app/core/rest/search.service.ts

```diff
@@ -44,7 +44,11 @@ export class SearchService {
   constructor(private http: HttpClient,
               private store: Store<AppState>,
               private homePageService: HomePageService) {
-    this.store.select(selectWorkspace).subscribe(workspace => this.workspace = workspace);
+    this.store.select(selectWorkspace)
+      .pipe(
+        filter(workspace => !isNullOrUndefined(workspace) && !isNullOrUndefined(workspace.organizationCode) && !isNullOrUndefined(workspace.projectCode))
```

Comment:
I'm just curious. Why do we need to use `isNullOrUndefined` here?

Reply:
I wanted to use `workspace && workspace.organizationCode && workspace.projectCode`, but it return string. Do you have any other idea?

Reply:
The typical trick in JavaScript is:
`!!workspace && !!workspace.organizationCode && !!workspace.projectCode`

Reply:
Or `Boolean(workspace && workspace.organizationCode && workspace.projectCode)`, should work the same.